syllabus.md

```diff
@@ -0,0 +1,32 @@
+# Syllabus
+
+The following topics relate to code quality.
+
```

Comment:
Common design patterns (in Go).

Reply:
The patterns I find interesting are listed below, I am not sure what you mean by this comment.